docs/rundown.rst

```diff
@@ -28,12 +28,67 @@ You can verify that you have a keypair with the following command:
 
 If the above command produces no output, or one line of output that says that a
 "trustdb" was created, then you do not have a secret key in your default
-keyring. If it produces output other than that, then you have a valid secret key
-and are ready to move on!
+keyring. Please see the section below titled "Adding a GPG key to AWX or Ansible Automation Controller."
+
+If it produces output other than that, then you have a valid secret key
+and are ready to move on to "How to Access the Ansible Sign CLI Utility."
+
+Adding a GPG key to AWX or Ansible Automation Controller:
+=========================================================
+
+In the command line, run the following commands:
+``$ gpg --list-keys``
+``$ gpg --export --armour <key fingerprint> > my_public_key.asc``
+
+
+In AWX/Automation Controller, click “Credentials."
+From there, click "Add."
+For “Name” name your credential something you will recognize.
+For “Credential Type” select  “GPG Public Key."
+
+Click "Browse" to navigate to and select the file that you created earlier. Finally, click the "Save" button to finish.
+
+This credential can now be selected in "Project" settings. Once selected, content verification will automatically take place on future project syncs.
+
+Vist `GPG`_ for more information regarding GPG keys.
+For more information regarding generating a GPG key pair, visit the `Red Hat "Enable Sysadmin" blog post`_.
 
 .. _GPG: https://www.gnupg.org/
 .. _Red Hat "Enable Sysadmin" blog post: https://www.redhat.com/sysadmin/creating-gpg-keypairs
 
+How to Access the Ansible Sign CLI Utility:
+===========================================
+
+Run the following command to install Ansible-Sign:
+``$ pip install ansible-sign ``
+
+Once it’s installed, run:
+``$ ansible-sign --help`` 
```

Comment:
I wonder if instead of `--help` we give the output of `--version` or something, if the goal here is just a self-check that the script installed correctly.

The output of `--version` is likely to be more stable (well, the version will change of course, but the format isn't likely to). This eliminates needing to update this document every time a new flag or top-level subcommand is added to the utility.

Reply:
I agree! That definitely makes sense. I have made that change!